File: filter_non_unique/filter_non_unique.rb
def filter_non_unique(list)
    list.select {|val| is_duplicate(list.count(val))}
end

def is_duplicate(val)
    val < 2
end


print filter_non_unique([1, 2, 2, 2, 3, 3, 4, 5, 9, 9])

File: filter_non_unique/filter_non_unique.py
from collections import Counter


class TestFilter:
    def filter_non_unique(self, lst):
        return [item for item, count in Counter(lst).items() if count == 1]


print TestFilter().filter_non_unique([1, 2, 2, 2, 3, 3, 4, 5, 9, 9])
print TestFilter().filter_non_unique_other([1, 2, 2, 2, 3, 3, 4, 5, 9, 9])
